cosmetics-web/app/helpers/responsible_persons/wizard/preselection_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ResponsiblePersons::Wizard::PreselectionHelper
+  def preselect_internal_reference_yes
+    (params[:notification].present? && params[:notification][:add_internal_reference] == "yes") ||
+      @notification.industry_reference.present?
```

Comment:
Suggetion: `dig` may be helpful here.
```suggestion
    params.dig(:notification, :add_internal_reference) == "yes" || @notification.industry_reference.present?
```